lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -402,6 +402,26 @@ def __OnPrimsChanged(self, notice, sender):
                 if hasattr(ctrl, 'refresh'):
                     ctrl.refresh()
 
+def connectionsCustomControlCreator(self, c):
```

Comment:
By Python convention, self should be reserved for the object argument on class methods.  Should call it aeTemplate, or similar.